pkg/lvgl/Makefile

```diff
@@ -0,0 +1,24 @@
+PKG_NAME=lvgl
+PKG_URL=https://github.com/littlevgl/lvgl
+PKG_VERSION=v6.1.1
```

Comment:
Version 6.1.2 was released 7 days ago and only includes bug fixes. Maybe you could bump it if that's okay with you.

Reply:
IMHO, that doesn't make any difference